lms/static/sass/course/instructor/_instructor_2.scss

```diff
@@ -134,6 +134,70 @@
       }
     }
   }
+  
+    .btn-blue {
+        @extend %btn-primary-blue;
+        margin-bottom: 0;
+        padding: ($baseline/2.5) ($baseline/2);
+        text-shadow: none;
+    }
+
+    // Custom File upload
+    .customBrowseBtn {
+        display: inline-block;
+        position: relative;
+        margin: ($baseline/2) 0 0;
+        width: 350px;
+        overflow: hidden;
+        vertical-align: bottom;
+
+        .enhanced-input-file {
+            @include border-radius(4px 0 0 4px);
```

Comment:
Correct. I tried to cleanup the "fancy file upload". There were discrepancies between Chrome and Firefox with respect to the file upload input type, which isn't natively style-able. Using the shadow DOM in Chrome, I was able to bring them generally closer in line.

I will be happy to clean this up too, as part of that other work.